---
output:
  word_document: default
  pdf_document:
    keep_tex: true
geometry: margin=1.0in
font-size: 11pt
header-includes:
  - \usepackage{helvet}
  - \renewcommand*\familydefault{\sfdefault}
  - \usepackage{setspace}
  - \doublespacing
  - \usepackage[left]{lineno}
  - \linenumbers
editor_options: 
  chunk_output_type: console
---

```{r, echo=FALSE}
library(tidyverse)
library(knitr)
library(here)
```

# Amplicon sequence variants should not replace operational taxonomic units in marker-gene data analysis


\vspace{20mm}

**Running title:** ASVs vs. OTUs

\vspace{20mm}

Patrick D. Schloss${^\dagger}$

\vspace{40mm}

${\dagger}$ To whom corresponsdence should be addressed:


\href{mailto:pschloss@umich.edu}{pschloss@umich.edu}

Department of Microbiology & Immunology

University of Michigan

Ann Arbor, MI 48109

\vspace{20mm}

**Observation Format**

\newpage


## Abstract (250 words)

## Importance (150 words)

\newpage

16S rRNA gene sequencing is a very powerful technique for describing and comparing microbial communities. Efforts to link 16S rRNA gene sequences to taxonomic levels based on distance thresholds go back to at least the 1990s. The distance-based thresholds that were developed and are now widely used (3%) were based on DNA-DNA hybridization approaches that are not as precise as genome sequencing. Instead, genome sequencing technologies have suggested that the widely used 3% distance threshold to operationally define bacterial taxa is too coarse. As an alternative to OTUs, amplicon sequencing variants (ASVs) have been proposed as a way to adopt the thresholds suggested by genome sequencing to microbial community analysis using 16S rRNA gene sequences. ASVs are a unit of microbial community inference that do not cluster sequences based on a distance-based threshold. However, most bacterial genomes have more than 1 copy of the rrn operon and those copies are not identical. Therefore, using too fine a threshold to identify OTUs creates the risk of splitting a single genome into multiple bins and using too broad of a threshold to define OTUs creates the risk of lumping together bacterial species into the same OTU. An example of both is seen in the comparison of Staphylococcus aureus (NCTC 8325) and S. epidermidis (ATCC 12228) where each genome has 5 copies of the 16S rRNA gene. The 10 copies of the 16S rRNA gene each have a different sequence and so if OTUs are defined based on ASVs, each genome would be split into 5 OTUs. Conversely, if the copies were clustered using a 3% distance threshold all 10 copies would cluster into the same OTU. The goal of this study was to quantify the risk of splitting a single genome into multiple bins and the risk of lumping together different bacterial species into the same bin.

```{r, echo=FALSE}
metadata <- read_tsv(here("data/references/genome_id_taxonomy.tsv"),
										 col_types = cols(.default = col_character()))

easv <- read_tsv(here("data/processed/rrnDB.easv.count_tibble"),
								col_types = cols(.default = col_character(),
																 count = col_integer()))

metadata_easv <- inner_join(metadata, easv, by=c("genome_id" = "genome")) %>%
	mutate(threshold = recode(threshold, "esv" = "0.000"),
				 threshold = as.numeric(threshold))

rrn_copy_number <- metadata_easv %>% 
	filter(region == "v19", threshold == 0) %>%
	group_by(species, genome_id) %>%
	summarize(n_rrn = sum(count), .groups="drop") %>%
	group_by(species) %>%
	summarize(min = min(n_rrn), max = max(n_rrn), median = median(n_rrn), N=n(), .groups="drop")
	
single_copy <- rrn_copy_number %>% filter(median == 1) %>% top_n(n=1, N)
max_copy <- rrn_copy_number %>% top_n(n=1, median) 

	
single_copy_N <- rrn_copy_number %>% filter(species == single_copy$species) %>% pull(N)
max_copy_N <- rrn_copy_number %>% filter(species == max_copy$species) %>% pull(N)
ecoli_copy <- rrn_copy_number %>% filter(species == "Escherichia coli")


rates <- metadata_easv %>%
	filter(threshold == 0) %>%
	group_by(region, genome_id) %>%
	summarize(n_rrns = sum(count), n_asvs = n_distinct(easv), .groups="drop") %>%
	group_by(region) %>%
	summarize(rate = sum(n_asvs)/sum(n_rrns), .groups="drop")

rate_v19 <- rates %>% filter(region == "v19") %>% pull(rate)
rate_v4 <- rates %>% filter(region == "v4") %>% pull(rate)
rate_v34 <- rates %>% filter(region == "v34") %>% pull(rate)
rate_v45 <- rates %>% filter(region == "v45") %>% pull(rate)


asvs_per_species <- metadata_easv %>%
	filter(threshold == 0 & region == "v19") %>%
	group_by(species) %>%
	summarize(n_asvs = n_distinct(easv), .groups="drop")

single_nasvs <- asvs_per_species %>% filter(species == single_copy$species) %>% pull(n_asvs)
ecoli_nasvs <- asvs_per_species %>% filter(species == "Escherichia coli") %>% pull(n_asvs)
```


To investigate the variation in the number of copies of the 16S rRNA gene per genome as well as the intragenomic variation among copies of the 16S rRNA gene, I obtained reference 16S rRNA sequences from the rrn copy number database (rrnDB; CITATION). Among the `r n_distinct(metadata$species)` species represented in the rrnDB there were `r nrow(metadata)` genomes. The median number of rrn operson per species ranged between `r single_copy$median` (e.g. `r single_copy$species`) and `r max_copy$median` (e.g. `r max_copy$species`) copies of the rrn operon. As the number of copies of the operon in a genome increased, the number of variants of the 16S rRNA gene in each genome also increased (**FIGURE**). On average, there were `r rate_v19` variants per copy of the full length 16S rRNA gene and an average of `r rate_v4`, `r rate_v34`, and `r rate_v45` variants when considering the V4, V3-V4, and V4-V5 regions of the gene, respectively. Although a species tended to have a consistent number of 16S rRNA gene copies per genome, the number of total variants increased with the number of genomes that were sampled (**FIGURE**). For example, `r single_copy$species` generally only had `r single_copy$median` copy of the gene per genome, but across the `r single_copy_N` genomes that have been sequenced there were `r single_nasvs` versions of the gene. Similarly, a E. coli genome typically had `r ecoli_copy$median` copies of the 16S rRNA gene with between `r ecoli_copy$min` and `r ecoli_copy$max` distinct full length sequences per genome. Across the `r ecoli_copy$N` E. coli genomes that have been sequenced, there were `r ecoli_nasvs` different variants of the gene. These observations highlight the risk of selecting a threshold for defining units of inference that is too narrow because it is possible to split a single genome into multiple units.

A method to avoid splitting a single genome into multiple units of inference is to cluster 16S rRNA gene sequences together that are similar. However, this also increases the risk of lumping together genes from different species that are similar to each other. Therefore, I assessed the impact of the threshold used to define clusters of 16S rRNA genes on the propopensity to lump species together and split genome apart. I identified the threshold where 90% of bacterial species would be represented by a single OTU. For full length 16S rRNA gene sequences, I found that at a threshold of XX%, 90% of the species would be represented by a single OTU. Similarly, thresholds of XX, XX, and XX% were observed for the V4, V3-V4, and V4-V5 regions. However, at these thresholds, multiple species could be represented by the same OTU. At the highest level of resolution, XX% of the species shared a 16S rRNA gene sequence variant with another species. Given the risk of splitting a genome into multiple OTUs is more biologically problematic than lumping species together, larger thresholds are advisable.

To provide a more nuanced approach to selecting a threshold, it would be useful to to quantify the sensitivity and specificity of characterizing bacterial species using OTUs defined at different thresholds. I created confusion matrices for multiple regions of the 16S rRNA gene: true positives were those cases where two ASVs were joined in the same OTU and the same species; true negatives were those cases where two ASVs from different OTUs came from different species; false positives were those ASVs that joined the same OTU, but were from different species; and false negatives were those ASVs that joined different OTUs, but were from the same species. By calculating the sensitivity and specificity for each threshold and each region of the 16S rRNA gene, I was able to constuct a receiver operator characteristic curve (ROC). Because the ROC curve represents a range of possible thresholds and sensitivities and specificities, I used two metrics to select the best threshold for defining an OTU. First, I identified the thresholds where the sensitivity and specificity were most similar to each other. For this criterion, the best thresholds were XX (V1-V9), XX (V4), XX (V3V4), and XX (V4V5). Second, I identified the threshold that resulted in the point on the ROC curve that was closest to perfect classification. For this criterion, the best thresholds were XX (V1-V9), XX (V4), XX (V3V4), and XX (V4V5). Surprisingly, these analyses revealed that thresholds near 3% balance the risks of splitting genomes into separate OTUs and lumping species into the same OTU.

The results of this analysis demonstrate that there is a significant risk of splitting single genomes into multiple bins if too fine of a threshold is applied to defining an OTU. An ongoing problem for amplicon-based studies is defining a meaningful taxonomic unit of inference. Since there is no consensus definition for a biological species concept, microbiologists must accept that how we have named bacterial species is biased and that taxonomic rules are not applied in a consistent manner. This makes it more challenging to attempt to fit a distance threshold to define an OTU definition that matches a set of species names. Furthermore, it is unlikely that the 16S rRNA gene evolves at the same rate across all bacterial lineages, which limits the biological interpretation of a common OTU definition. At best, a distance-based definition of a taxonomic unit is operational. There is general agreement in bacterial systematics that to classify something to a bacterial species, you need phenotypic and genome sequence data (CITATION). We are asking too much of a short section of a bacterial genome to be able to differentiate between species. It is difficult to defend a unit of inference that would split a single genome into multiple taxonomic units. It is not biologically plausible to entertain the possability that parts of a genome would have different ecologies. Although there are multiple reasons that proponents of ASVs encourage their use, the significant risk of splitting genomes is too high to warrant their use.


**Materials and Methods. (i) Data availability.** The 16S rRNA gene sequences used in this study were  obtained from the *rrn*DB (https://rrndb.umms.med.umich.edu; version 5.6, released November 8, 2019). At the time of submission, this is the most current version of the database. The *rrn*DB obtained the curated 16S rRNA gene sequences from the KEGG database, which ultimately obtained them from NCBI's non-redundant RefSeq database. The *rrn*DB provides downloadable versions of the sequences with their taxonomy as determined using the naive Bayesian classifier trained on the RDP reference taxonomy. For some genomes this resulted in multiple classifications since a genome's 16S rRNA gene sequences were not identical. Instead, I mapped the RefSeq accession number for each genome in the database to obtain a single taxonomy for each genome. Because strain names were not consistently given to genomes across bacterial species, the strain level designations were ignored.

**(ii) Definition of regions within 16S rRNA gene.** The full length 16S rRNA gene sequences were aligned to a SILVA reference alignment of the 16S rRNA gene (v138) using the mothur software package (v. 1.XX). Regions of the 16S rRNA gene were selected because of their use in the microbial ecology literature. Full length sequences corresponded to *E. coli* positions XX through XXXX, V4 to positions XXX through XXX, V3-V4 to positions XXX through XXX, and V4-V5 to positions XXX through XXX.

**(iii) Controlling for uneven sampling of genomes by species.** Because of the uneven distribution of genome sequences across species, for the analysis of splitting genomes and lumping species I randomly selected one genome for each species. The random selection was repeated 100 times. Analyses based on this randomization report the median of the 100 randomizations. The intraquartile range between randomizations was typically less than XXXX. Because it was so small, confidence intervals are not included in Figure 2.

**(iv) Reproducible data analysis.** The code to perform the analysis in this manuscript and its hisotry are available as a git-based version control repository on GitHub (https://github.com/pschloss/Schloss_rrnAnalysis_XXXX_2020). The analysis can be regenerated using a GNU Make-based workflow that made use of built-in bash tools (v. 3.2.57), mothur (v. 1.XX), and R (v. 4.X.X). Within R, I used the tidyverse (v. 4.X.X), data.table (v. 4.X.X), Rcpp (v. 4.X.X), furrr (v. 4.X.X), and rmarkdown (v. 4.X.X) packages. The conception and development of this analysis is available as a playlist on the Riffomonas YouTube channel (https://www.youtube.com/playlist?list=PLmNrK_nkqBpKY3SZiivlIGvcLX-KHmfR8).

**Acknowledgements.** I am grateful to Robert Hein and Thomas Schmidt who maintain the rrnDB for their help in understanding the curation of the database and for making the 16S rRNA gene sequences and related metadata publicly available. I am also grateful to community members who watched the serialized version of this analysis on YouTube and provided their suggestions and questions.

This work was supported, in part, through grants from the NIH to PDS (P30DK034933, U01AI124255, and R01CA215574).

\newpage

## References


\newpage

## Figures
