---
output:
  word_document: default
  pdf_document:
    keep_tex: true
geometry: margin=1.0in
font-size: 11pt
header-includes:
  - \usepackage{helvet}
  - \renewcommand*\familydefault{\sfdefault}
  - \usepackage{setspace}
  - \doublespacing
  - \usepackage[left]{lineno}
  - \linenumbers
editor_options:
  chunk_output_type: console
bibliography: references.bib
csl: asm.csl
---

```{r, echo=FALSE}
library(tidyverse)
library(knitr)
library(here)

opts_chunk$set(echo = FALSE)

inline_hook <- function(x){

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps){
			# treat as integer
			formatted <- format(x, digits=0, big.mark = ",")

		} else {
			# treat as floating point number
			formatted <- format(x, digits=2, nsmall=2, big.mark = ",")
		}
	} else {
		formatted <-x
	}

	paste0("**", formatted, "**")


}


knit_hooks$set(inline=inline_hook)

format_pct <- function(x, my_digits=1){

	format(x, digits=my_digits, nsmall=my_digits)

}
```

# Amplicon sequence variants artificially split bacterial genomes into separate units of inference


\vspace{20mm}

**Running title:** ASVs artificially split bacterial genomes

\vspace{20mm}

Patrick D. Schloss${^\dagger}$

\vspace{40mm}

${\dagger}$ To whom corresponsdence should be addressed:


\href{mailto:pschloss@umich.edu}{pschloss@umich.edu}

Department of Microbiology & Immunology

University of Michigan

Ann Arbor, MI 48109

\vspace{20mm}

**Observation Format**

\newpage

```{r}
## Paragraph 1
metadata <- read_tsv(here("data/references/genome_id_taxonomy.tsv"),
										 col_types = cols(.default = col_character()))

easv <- read_tsv(here("data/processed/rrnDB.easv.count_tibble"),
								col_types = cols(.default = col_character(),
																 count = col_integer()))

metadata_easv <- inner_join(metadata, easv, by=c("genome_id" = "genome")) %>%
	mutate(threshold = recode(threshold, "esv" = "0.000"),
				 threshold = as.numeric(threshold))

rrn_copy_number <- metadata_easv %>%
	filter(region == "v19", threshold == 0) %>%
	group_by(species, genome_id) %>%
	summarize(n_rrn = sum(count), .groups="drop") %>%
	group_by(species) %>%
	summarize(min = min(n_rrn), max = max(n_rrn), median = median(n_rrn), N=n(), .groups="drop")

single_copy <- rrn_copy_number %>% filter(median == 1) %>% top_n(n=1, N)
max_copy <- rrn_copy_number %>% top_n(n=1, median)


single_copy_N <- rrn_copy_number %>% filter(species == single_copy$species) %>% pull(N)
max_copy_N <- rrn_copy_number %>% filter(species == max_copy$species) %>% pull(N)
ecoli_copy <- rrn_copy_number %>% filter(species == "Escherichia coli")


rates <- metadata_easv %>%
	filter(threshold == 0) %>%
	group_by(region, genome_id) %>%
	summarize(n_rrns = sum(count), n_asvs = n_distinct(easv), .groups="drop") %>%
	group_by(region) %>%
	summarize(rate = sum(n_asvs)/sum(n_rrns), .groups="drop")

rate_v19 <- rates %>% filter(region == "v19") %>% pull(rate)
rate_v4 <- rates %>% filter(region == "v4") %>% pull(rate)
rate_v34 <- rates %>% filter(region == "v34") %>% pull(rate)
rate_v45 <- rates %>% filter(region == "v45") %>% pull(rate)


asvs_per_species <- metadata_easv %>%
	filter(threshold == 0 & region == "v19") %>%
	group_by(species) %>%
	summarize(n_asvs = n_distinct(easv), .groups="drop")

single_nasvs <- asvs_per_species %>% filter(species == single_copy$species) %>% pull(n_asvs)
ecoli_nasvs <- asvs_per_species %>% filter(species == "Escherichia coli") %>% pull(n_asvs)

## Paragraph 2
thresholds <- read_tsv(here("data/processed/thresholds_for_single_otu.tsv"),
											 col_types=cols(region = col_character(),
											 							 .default = col_double())) %>%
								filter(n_genomes > 100)

threshold_v19 <- thresholds %>% filter(n_rrns == 7 & region == "v19") %>% pull(threshold) * 100
threshold_v4 <- thresholds %>% filter(n_rrns == 7 & region == "v4") %>% pull(threshold) * 100
threshold_v34 <- thresholds %>% filter(n_rrns == 7 & region == "v34") %>% pull(threshold) * 100
threshold_v45 <- thresholds %>% filter(n_rrns == 7 & region == "v45") %>% pull(threshold) * 100


three_v19 <- thresholds %>% filter(threshold <= 0.03 & region == "v19") %>% tail(n=1) %>% pull(n_rrns)
three_v4 <- thresholds %>% filter(threshold <= 0.03 & region == "v4") %>% tail(n=1) %>% pull(n_rrns)
three_v34 <- thresholds %>% filter(threshold <= 0.03 & region == "v34") %>% tail(n=1) %>% pull(n_rrns)
three_v45 <- thresholds %>% filter(threshold <= 0.03 & region == "v45") %>% tail(n=1) %>% pull(n_rrns)


## Paragraph 3
lump_split <- read_tsv(here("data/processed/lumped_split_rate.tsv"),
											 col_types = cols(region = col_character(),
											 								 .default = col_double()))

lump_v19 <- lump_split %>% filter(threshold ==0 &region == "v19") %>% pull(lump_rate) * 100
lump_v4 <- lump_split %>% filter(threshold ==0 &region == "v4") %>% pull(lump_rate) * 100
lump_v34 <- lump_split %>% filter(threshold ==0 &region == "v34") %>% pull(lump_rate) * 100
lump_v45 <- lump_split %>% filter(threshold ==0 &region == "v45") %>% pull(lump_rate) * 100

lump3_v19 <- lump_split %>% filter(threshold ==0.03 & region == "v19") %>% pull(lump_rate) * 100
lump3_v4 <- lump_split %>% filter(threshold ==0.03 & region == "v4") %>% pull(lump_rate) * 100
lump3_v34 <- lump_split %>% filter(threshold ==0.03 & region == "v34") %>% pull(lump_rate) * 100
lump3_v45 <- lump_split %>% filter(threshold ==0.03 & region == "v45") %>% pull(lump_rate) * 100
```

## Abstract

Amplicon sequencing variants (ASVs) have been proposed as an alternative to operational taxonomic units (OTUs) for analyzing microbiomes. ASVs have grown in popularity, in part, because of a desire to reflect a more refined level of taxonomy because they do not cluster sequences based on a distance-based threshold. However, ASVs and the use of overly narrow thresholds to identify OTUs increases the risk of splitting a single genome into separate clusters. I addressed this problem by analyzing the intragenomic variation of 16S rRNA genes from the bacterial genomes represented in the rrn copy number database, which contained `r nrow(metadata)` genomes from `r n_distinct(metadata$species)` species. The analysis confirmed the severity of this risk. As the number of copies of the 16S rRNA gene increased in a genome, the number of ASVs also increased. For full length 16S rRNA genes, there was an average of `r rate_v19` ASVs per copy of the 16S rRNA gene. Among genomes with 7 copies of the 16S rRNA, such as *E. coli*, it was necessary to use a distance threshold of `r format_pct(threshold_v19)`% to cluster the ASVs from the same genome into a single OTU. This research highlights the risk of splitting a single bacterial genome into separate clusters when ASVs are used to analyze 16S rRNA gene sequence data. Although there is also a risk of clustering different species into the same OTU, the effects of those risks on biological inferences are less than those from artificially splitting a genome into separate ASVs and OTUs.


## Importance

16S rRNA gene sequencing has propelled significant interest into host-associated and environmental microbiomes. There is a tension between trying to classify 16S rRNA gene sequences to increasingly lower taxonomic levels and the reality that those levels were defined using more sequence and physiological information than is available from a fragment of the 16S rRNA gene. Furthermore, naming of bacterial taxa reflects the biases of those who name them. One motivation for the recent push to adopt ASVs in place of OTUs in microbiome analyses is to allow researchers to perform their analyes at the finest possible level that reflects species-level taxonomy. The current research is significant because it quantifies the risk of artificially splitting bacterial genomes into separate clusters. Far from providing a better represenation of microbiomes, the ASV approach could lead to conflicting inferences about the ecology of different clusters from the same genome.



\newpage

16S rRNA gene sequencing is a powerful technique for describing and comparing microbial communities [@Lane1985]. Efforts to link 16S rRNA gene sequences to taxonomic levels based on distance thresholds go back to at least the 1990s. The distance-based thresholds that were developed and are now widely used (3%) were based on DNA-DNA hybridization approaches that are not as precise as genome sequencing [@Stackebrandt1994; @Goris2007]. Instead, genome sequencing technologies have suggested that the widely used 3% distance threshold to operationally define bacterial taxa is too coarse [@RodriguezR2018; @Stackebrandt2006; @Edgar2018]. As an alternative to OTUs, amplicon sequencing variants (ASVs) have been proposed as a way to adopt the thresholds suggested by genome sequencing to microbial community analysis using 16S rRNA gene sequences [@Edgar2016; @Amir2017; @Callahan2016; @Eren2014]. ASVs are a unit of microbial community inference that do not cluster sequences based on a distance-based threshold [@Callahan2017]. However, most bacterial genomes have more than 1 copy of the rrn operon and those copies are not identical [@Pei2010; @Sun2013]. Therefore, using too fine a threshold to identify OTUs creates the risk of splitting a single genome into multiple bins and using too broad of a threshold to define OTUs creates the risk of lumping together bacterial species into the same OTU. An example of both is seen in the comparison of *Staphylococcus aureus* (NCTC 8325) and *S. epidermidis* (ATCC 12228) where each genome has 5 copies of the 16S rRNA gene. The 10 copies of the 16S rRNA gene each have a different sequence and so if OTUs are defined based on ASVs, each genome would be split into 5 OTUs. Conversely, if the copies were clustered using a 3% distance threshold all 10 copies would cluster into the same OTU. The goal of this study was to quantify the risk of splitting a single genome into multiple bins and the risk of lumping together different bacterial species into the same bin.

To investigate the variation in the number of copies of the 16S rRNA gene per genome as well as the intragenomic variation among copies of the 16S rRNA gene, I obtained reference 16S rRNA sequences from the rrn copy number database (rrnDB)[@Stoddard2014]. Among the `r n_distinct(metadata$species)` species represented in the rrnDB there were `r nrow(metadata)` genomes. The median number of rrn operson per species ranged between `r single_copy$median` (e.g., *`r single_copy$species`*) and `r max_copy$median` (*`r max_copy$species`*) copies of the rrn operon. As the number of copies of the operon in a genome increased, the number of variants of the 16S rRNA gene in each genome also increased. On average, there were `r rate_v19` variants per copy of the full length 16S rRNA gene and an average of `r rate_v4`, `r rate_v34`, and `r rate_v45` variants when considering the V4, V3-V4, and V4-V5 regions of the gene, respectively. Although a species tended to have a consistent number of 16S rRNA gene copies per genome, the number of total variants increased with the number of genomes that were sampled (**Figure 1**). For example, *`r single_copy$species`* generally only had `r single_copy$median` copy of the gene per genome, but across the `r single_copy_N` genomes that have been sequenced there were `r single_nasvs` versions of the gene. Similarly, a *E. coli* genome typically had `r ecoli_copy$median` copies of the 16S rRNA gene with between `r ecoli_copy$min` and `r ecoli_copy$max` distinct full length sequences per genome. Across the `r ecoli_copy$N` *E. coli* genomes that have been sequenced, there were `r ecoli_nasvs` different variants of the gene. These observations highlight the risk of selecting a threshold for defining units of inference that is too narrow because it is possible to split a single genome into multiple units.

A method to avoid splitting a single genome into multiple units of inference is to cluster together similar 16S rRNA gene sequences. Therefore, I assessed the impact of the distance threshold used to define clusters of 16S rRNA genes on the propensity to split a genome into separate clusters. I observed that as the number of copies of the *rrn* operon increased, the distance threshold required to reduce the ASVs in each genome to a single OTU increased (Figure 1). Among species with 7 copies of the *rrn* operon (e.g., *E. coli*), I found that a threshold of `r format_pct(threshold_v19)`% was required to reduce full length ASVs to a single OTU in 95% of the species. Similarly, thresholds of `r format_pct(threshold_v4)`, `r format_pct(threshold_v34)`, and `r format_pct(threshold_v45)`% were required for the V4, V3-V4, and V4-V5 regions, respectively. But, if a 3% distance threshold was used, then ASVs from genomes containing fewer than `r three_v19`, `r three_v4`, `r three_v34`, and `r three_v45` copies of the *rrn* operon would reliably be clustered into a single OTU for ASVs from the V1-V9, V4, V3-V4, and V4-V5 regions, respectively. Consequently, these results demonstrate that broad thresholds must be used to avoid splitting different operons from the same genome into separate clusters.

At broad thresholds multiple species could be represented by the same OTU (**Figure 2**). Using ASVs, `r format_pct(lump_v19)`% of the species shared a 16S rRNA gene sequence variant with another species when considering full length sequences and `r format_pct(lump_v4)`, `r format_pct(lump_v34)`, and `r format_pct(lump_v45)`% when considering the V4, V3-V4, and V4-V5 regions, respectively. At the commonly used 3% threshold, `r format_pct(lump3_v19)`% of the species shared an OTU when considering full length sequences and `r format_pct(lump3_v4)`, `r format_pct(lump3_v34)`, and `r format_pct(lump3_v45)`% when considering the V4, V3-V4, and V4-V5 regions, respectively. Considering that species designations are unevenly applied and reflect multiple biases, the risk of splitting a genome into multiple OTUs more problematic than clustering species together. Therefore, larger thresholds are advisable.

The results of this analysis demonstrate that there is a significant risk of splitting single genomes into multiple bins if too fine of a threshold is applied to defining an OTU. An ongoing problem for amplicon-based studies is defining a meaningful taxonomic unit of inference [@Callahan2017; @Schloss2011; @Johnson2019]. Since there is no consensus definition for a biological species concept [@Staley2006; @Oren2013], microbiologists must accept that how we have named bacterial species is biased and that taxonomic rules are not applied in a consistent manner (e.g., [@Baltrus2016]). This makes it more challenging to attempt to fit a distance threshold to define an OTU definition that matches a set of species names [@Konstantinidis2005]. Furthermore, the 16S rRNA gene does not evolve at the same rate across all bacterial lineages [@Schloss2011], which limits the biological interpretation of a common OTU definition. A distance-based definition of a taxonomic unit based on 16S rRNA gene or full genome sequences is, at best, operational and not grounded in biological theory [@Schloss2011; @Barco2020; @Parks2020; @Yarza2014]. There is general agreement in bacterial systematics that to classify something to a bacterial species, you need phenotypic and genome sequence data [@Staley2006; @Oren2013; @Baltrus2016]. We are asking too much of a short section of a bacterial genome to be able to differentiate between species. It is difficult to defend a unit of inference that would split a single genome into multiple taxonomic units. It is not biologically plausible to entertain the possibility that parts of a genome would have different ecologies. Although there are multiple reasons that proponents of ASVs encourage their use, the significant risk of splitting genomes is too high to warrant their use.


**Materials and Methods. (i) Data availability.** The 16S rRNA gene sequences used in this study were obtained from the *rrn*DB (https://rrndb.umms.med.umich.edu; version 5.6, released November 8, 2019) [@Stoddard2014]. At the time of submission, this is the most current version of the database. The *rrn*DB obtained the curated 16S rRNA gene sequences from the KEGG database, which ultimately obtained them from NCBI's non-redundant RefSeq database. The *rrn*DB provides downloadable versions of the sequences with their taxonomy as determined using the naive Bayesian classifier trained on the RDP reference taxonomy. For some genomes this resulted in multiple classifications since a genome's 16S rRNA gene sequences were not identical. Instead, I mapped the RefSeq accession number for each genome in the database to obtain a single taxonomy for each genome. Because strain names were not consistently given to genomes across bacterial species, the strain level designations were ignored.

**(ii) Definition of regions within 16S rRNA gene.** The full length 16S rRNA gene sequences were aligned to a SILVA reference alignment of the 16S rRNA gene (v138) using the mothur software package (v. 1.XX) [@Schloss2009; @Quast2012]. Regions of the 16S rRNA gene were selected because of their use in the microbial ecology literature. Full length sequences corresponded to *E. coli* positions XX through XXXX, V4 to positions XXX through XXX, V3-V4 to positions XXX through XXX, and V4-V5 to positions XXX through XXX.

**(iii) Controlling for uneven sampling of genomes by species.** Because of the uneven distribution of genome sequences across species, for the analysis of splitting genomes and lumping species I randomly selected one genome for each species. The random selection was repeated 100 times. Analyses based on this randomization report the median of the 100 randomizations. The intraquartile range between randomizations was typically less than XXXX. Because it was so small, confidence intervals are not included in Figure 2.

**(iv) Reproducible data analysis.** The code to perform the analysis in this manuscript and its hisotry are available as a git-based version control repository on GitHub (https://github.com/pschloss/Schloss_rrnAnalysis_XXXX_2020). The analysis can be regenerated using a GNU Make-based workflow that made use of built-in bash tools (v. 3.2.57), mothur (v. 1.XX), and R (v. 4.X.X). Within R, I used the tidyverse (v. 4.X.X), data.table (v. 4.X.X), Rcpp (v. 4.X.X), furrr (v. 4.X.X), and rmarkdown (v. 4.X.X) packages. The conception and development of this analysis is available as a playlist on the Riffomonas YouTube channel (https://www.youtube.com/playlist?list=PLmNrK_nkqBpKY3SZiivlIGvcLX-KHmfR8).

**Acknowledgements.** I am grateful to Robert Hein and Thomas Schmidt who maintain the rrnDB for their help in understanding the curation of the database and for making the 16S rRNA gene sequences and related metadata publicly available. I am also grateful to community members who watched the serialized version of this analysis on YouTube and provided their suggestions and questions.

This work was supported, in part, through grants from the NIH to PDS (P30DK034933, U01AI124255, and R01CA215574).

\newpage

## References

\setlength{\parindent}{-0.25in}
\setlength{\leftskip}{0.25in}
\noindent

<div id="refs"></div>

\setlength{\parindent}{0in}
\setlength{\leftskip}{0in}

\newpage

\includegraphics{../figures/copy_number_threshold_plot.pdf}

**Figure 1. The distance threshold required to prevent the splitting of genomes into multiple OTUs increases as the number of *rrn* operons in the genome increases.** Each line represents the median distance threshold for each region of the 16S rRNA gene that is required for 95% of the species with the indicated numbrer of *rrn* operons to cluster their ASVs to a single OTU. The median distance threshold was calculated across 100 randomizations in which one genome was sampled from each species. Only those number of *rrn* operons that were found in more than 100 species are included.

\newpage

\includegraphics{../figures/lump_split.pdf}

**Figure 2. As the distance threshold used to define an OTU increases, the fraction of genomes split into separate OTUs decreases while the fraction of species that are merged into the same OTU increases.** These data represent the median fractions for both measurements across 100 randomizations. In each randomization, one genome was sampled from each species.



\newpage

\includegraphics{../figures/esv_rate.pdf}

**Figure S1. The ratio of number of distinct ASVs per copy of the *rrn* operon increases for a species as the number of genomes sampled increases.** Each point represents a different species and is shaded to be 80% transparent so that when points overlap they become darker. The blue line represents a smoothed fit through the data.
