% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1.0in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage{helvet}
\renewcommand*\familydefault{\sfdefault}
\usepackage{setspace}
\doublespacing
\usepackage[left]{lineno}
\linenumbers
\ifluatex
  \usepackage{selnolig}  % disable illegal ligatures
\fi

\author{}
\date{\vspace{-2.5em}}

\begin{document}

\hypertarget{amplicon-sequence-variants-should-not-replace-operational-taxonomic-units-in-marker-gene-data-analysis}{%
\section{Amplicon sequence variants should not replace operational
taxonomic units in marker-gene data
analysis}\label{amplicon-sequence-variants-should-not-replace-operational-taxonomic-units-in-marker-gene-data-analysis}}

\vspace{20mm}

\textbf{Running title:} ASVs vs.~OTUs

\vspace{20mm}

Patrick D. Schloss\({^\dagger}\)

\vspace{40mm}

\({\dagger}\) To whom corresponsdence should be addressed:

\href{mailto:pschloss@umich.edu}{pschloss@umich.edu}

Department of Microbiology \& Immunology

University of Michigan

Ann Arbor, MI 48109

\vspace{20mm}

\textbf{Observation Format}

\newpage

\hypertarget{abstract-250-words}{%
\subsection{Abstract (250 words)}\label{abstract-250-words}}

\hypertarget{importance-150-words}{%
\subsection{Importance (150 words)}\label{importance-150-words}}

\newpage

\hypertarget{introduction}{%
\subsection{Introduction}\label{introduction}}

\begin{itemize}
\item
  16S rRNA gene sequencing is a very powerful technique for describing
  and comparing microbial communities
\item
  How do we analyze them (classification, clustering)?
\item
  What has changed in recent years? ASVs
\item
  Efforts to link 16S rRNA gene sequences to taxonomic levels based on
  distance thresholds go back a long way
\item
  ESVs/ASVs have been an attempt to adopt the thresholds suggested by
  genome sequencing to microbial community analysis using 16S rRNA gene
  sequences
\item
  Most bacterial genomes have more than 1 copy of the rrn operon and
  those copies are not identical
\item
  Using too fine a threshold to create taxonomic groups runs risk of
  splitting single genome into multiple bins
\item
  For example, E. coli K-12 has 7 copies of the 16S rRNA gene with 5
  variants
\item
  Using too broad a threshold to define ASVs or OTUs risks lumping
  together bacterial species into the same grouping
\item
  For example, B. cereus, thuringiensis, anthracis share the same 16S
  rRNA gene sequences
\item
  Goal of this study
\end{itemize}

\newpage

\hypertarget{results}{%
\subsection{Results}\label{results}}

\begin{itemize}
\tightlist
\item
  ESVs/ASVs

  \begin{itemize}
  \tightlist
  \item
    copy number varies by taxonomy
  \item
    more copies, more variants per genome
  \item
    full length sequences have more variants than sub-regions
  \item
    as more sequences are added to a species, the number of variants
    increases
  \end{itemize}
\item
  OTUs

  \begin{itemize}
  \tightlist
  \item
    increasing a threshold decreases the number of variants
  \item
    this limits the splitting of a single genome into multiple bins
  \item
    this increases the lumping of species into single bin
  \end{itemize}
\end{itemize}

\newpage

\hypertarget{conclusions}{%
\subsection{Conclusions}\label{conclusions}}

\begin{itemize}
\item
  Briefly synthesize results

  \begin{itemize}
  \tightlist
  \item
    Unlikely that the unit of inference should be an ASV
  \end{itemize}
\item
  No biological argument to split a genome into multiple bins
\item
  This analysis has allowed some splitting to balance with lumping
\item
  To reduce splitting further, you would need larger thresholds
\item
  There is general agreement in the field that if you want to classify
  something to a bacterial species, you need more than the 16S rRNA gene
\item
  Furthermore, using only a few hundred bases of that gene are even more
  limited.
\item
  We are asking too much of a short section of sequence
\item
  Surprisingly, 3\% performs pretty well for an operational definition
  that limits splitting of bacterial genomes and avoiding the lumping of
  bacterial species
\end{itemize}

\newpage

\hypertarget{materials-and-methods}{%
\subsection{Materials and Methods}\label{materials-and-methods}}

\begin{itemize}
\tightlist
\item
  rrnDB
\item
  NCBI taxonomy
\item
  R and R packages
\item
  GitHub / YouTube
\end{itemize}

\newpage

\hypertarget{acknowledgements}{%
\subsection{Acknowledgements}\label{acknowledgements}}

\newpage

\hypertarget{figures}{%
\subsection{Figures}\label{figures}}

\end{document}
